### ISSUE TITLE
old and new version blob simultaneously exist new Modify the code

### ISSUE THREAD

**Issue by Hello-xBugs** [djhohnstein/SharpChrome#8]
Many chrome is upate to 80+ ,not direct installation，will have old cookie and new cookie blob in Cookies DB, thanks to update the Code

**Comment by djhohnstein** [comment]
Yep, I'm aware of the issue. SharpChrome incorporated in SharpDPAPI should let you retrieve that stuff, but if you wanted to quick-hack fix it, the lines you would need to change are here: 

https://github.com/djhohnstein/SharpChromium/blob/master/ChromiumCredentialManager.cs#L136

All you'd need to do is check output, and if null or empty, use DPAPI decryption instead.
